raft/constants.go

```diff
@@ -1,11 +1,18 @@
 package raft
 
+import (
+	etcdRaft "github.com/coreos/etcd/raft"
+)
+
 const (
 	//protocolName           = "raft"
 	//protocolVersion uint64 = 0x01
 
 	//raftMsg = 0x00
 
+	minterRole = etcdRaft.LEADER
```

Comment:
I am not fan of re-declaring an etcd constant as a constant here (why not just using the constant itself?)

Reply:
@amalrajmani only applied the code on quorum-geth-1.9.7. we need in the future to re-do this using the right StateType API on this issue: https://github.com/ConsenSysQuorum/quorum/issues/189